apps/www/.storybook/main.ts

```diff
@@ -0,0 +1,19 @@
+import type { StorybookConfig } from "@storybook/nextjs"
+
+const config: StorybookConfig = {
+  stories: ["../stories/**/*.mdx", "../stories/**/*.stories.@(js|jsx|ts|tsx)"],
```

Comment:
```suggestion
  stories: ["../stories/**/*.mdx", "../stories/**/*.stories.@(js|jsx|ts|tsx)", "../registry/stories/**/*.stories.@(js|jsx|ts|tsx)"],
```

I had to add this to get the components to show up in the storybook.